tasks/tasks.py

```diff
@@ -415,3 +414,105 @@ def _handle_test_exception(self, exc):
 
 class RunADTests(RunPytest):
     action_name = 'ad'
+
+
+class RunMultiDomainPytest(JobTask):
```

Comment:
Instead of defining a new class, you can modify the existing RunPytest class:
- in `__init__`, add the new parameter `trusted_domain=False` just before kwargs
- in `__init__`, add this line: `self.trusted_domain = trusted_domain`
- in `_before`, add this line `"trusted_domain": self.trusted_domain,` in the call to `create_file_from_template`